python/desc/twinkles/twinkles_sky.py

```diff
@@ -106,11 +103,11 @@ def __init__(self,
         # Lists of component phosim Instance Catalogs and CatalogDBObjects
         # Stars
         self.compoundStarDBList = [StarObj, CepheidStarObj]
-        self.compoundStarICList = [PhoSimCatalogPoint, PhoSimCatalogPoint]
+        self.compoundStarICList = [TwinklesPhoSimCatalogPoint, TwinklesPhoSimCatalogPoint]
```

Comment:
I suggest @rbiswas4 should merge #373.  I will then pull down the new master, rebase, and fix any conflicts in this PR.  You can then re-review it before I merge.  Sound good?